migtests/tests/pg/sequences/pg_sequences_automation.sql

```diff
@@ -214,4 +214,39 @@ WITH region_list AS (
                 amount[1 + mod(n, array_length(amount, 1))], 
```

Comment:
Is this test already running in live migration & offline with BETA_FAST_DATA_EXPORT?

Reply:
Only Offline with Beta_fast mode, we don't run pg_sequences test for live-migration currently